e2e/tests/pages/components/back-link.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('/components/back-link/', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/components/back-link/')
+  })
+
+  test('has the correct title tag', async ({ page }) => {
+    await expect(page).toHaveTitle('Back link – MOD.UK Design System')
+  })
+
+  test.describe('code snippet', () => {
+    test.describe('@visual-regression', () => {
+      test('the outer box matches the screenshot on focus', async ({ page }) => {
+        const codeSnippet = page.locator('.guidance-code-snippet').first()
+        await codeSnippet.locator('pre').focus()
+
+        await expect(codeSnippet).toHaveScreenshot('copy-snippet-box-focus.png')
+      })
+
+      test('the copy button matches the screenshot on focus', async ({ browserName, page }) => {
+        // https://github.com/microsoft/playwright/issues/18901
+        test.skip(browserName === 'webkit', 'WebKit does not support navigator.clipboard on Linux')
```

Comment:
Had to skip this in webkit as per the comment – assume (or hope) it works in Safari running on an actual Mac though.

Reply:
Tested on Safari Version 16.1 works fine